taar/recommenders/local_recommender.py

```diff
@@ -28,7 +28,7 @@ class LocalRecommender:
     """
     def __init__(self):
         self.model = None
```

Comment:
Since you're redefining this the line below, you're good to remove the `self.model = None` line here.